end2end/tests/Workflow Editor - Creating and Editing events.spec.ts

```diff
@@ -0,0 +1,131 @@
+import{test, expect} from "../fixtures/MyCustomWorkflowFixtures"
+
+test.use({
+  ignoreHTTPSErrors: true
+});
+
+
+test.beforeAll(async ({workflowData})=>{
+  let randNum = Math.random();
+  workflowData.uniqueText = `New Event ${randNum}`;
+  workflowData.uniqueDescr = `Adding description ${randNum}`;
+
+})
+
+//Create a New Event
+test ('CreateEvent', async ({ page, workflowData}, testinfo) => {
+ await page.goto('https://host.docker.internal/Test_Request_Portal/admin/?a=workflow');
+ await page.goto('https://host.docker.internal/Test_Request_Portal/admin/?a=workflow&workflowID=1');
+ 
+//Click on the Requestor 
+  await expect(page.getByText('Return to Requestor')).toBeVisible();
+  await page.getByText('Return to Requestor').click();
+
+//Add a new event
+  await expect(page.getByRole('button', { name: 'Add Event' })).toBeVisible();
+  await page.getByRole('button', { name: 'Add Event' }).click();
+
+
+  // Wait for the Create Event page to load
+    await expect(page.getByRole('button', { name: 'Save' })).toBeVisible();
+    await page.getByRole('button', { name: 'Create Event' }).click();
+    
+  //Enter Data for New Event
+
+
+    //await page.getByRole('button', { name: 'Create Event' }).click();
+    await page.getByLabel('Event Name:').click();
+    await page.getByLabel('Event Name:').fill(workflowData.uniqueText);
+    await page.getByLabel('Short Description: Notify').fill(workflowData.uniqueDescr);
+    await page.getByText('Notify Requestor Email: Notify Next Approver Email: Notify Group: None2911 TEST').click();
+    await page.getByLabel('Notify Requestor Email:', { exact: true }).check();
+    await page.getByLabel('Notify Next Approver Email:', { exact: true }).check();
+    await page.getByLabel('Notify Group:', { exact: true }).selectOption('206');
+   
+    await expect(page.getByRole('button', { name: 'Save' })).toBeVisible();
+  
+  //Save Event
+  await page.getByRole('button', { name: 'Save' }).click();
+   
+  await expect(page.getByRole('button', { name: 'Remove Action' })).toBeVisible();
+ 
+  //Verify present
+  let eventTitle = `Email - ${workflowData.uniqueDescr}`;
+  await expect(page.locator('#stepInfo_3')).toContainText(eventTitle);
+
+  await page.getByLabel('Close Modal').click();
+
+  const screenshot = await page.screenshot()
+  await testinfo.attach('screenshot', { body: screenshot, contentType: 'image/png' });

```

Comment:
The screenshot should be taken prior to closing the modal in order to show the new events.